modules/ROOT/pages/manage-partner-certificates.adoc

```diff
@@ -0,0 +1,85 @@
+= Managing Partner and Third-Party Connection Certificates
+:page-aliases: manage-as2-partner-certificates.adoc
+
+Anypoint Partner Manager supports the following types of partner and third-party-connection certificates:
+
+[%header%autowidth.spread]
+|===
+|Certificate Type |Description | Used by
+| Public certificate | Certificate generated by a certificate  (CA) a| 
+* AS2 Send to Partners endpoints
+* HTTPS Send to Partners endpoints
+| Identity key certificate a| Private key used for authentication a| 
+* SFTP Receive from Partners endpoints, when the authentication type is `Identity Key` or `Basic and Identity Key` 
+* SFTP Send to Partners endpoints, when the authentication type is `Identity Key` or `Basic and Identity Key` 
+|===
+
+== Add a Certificate
+
+To add a partner or third-party-connection certificate:
+
+. In the sidebar, access the *Partners* page.
+. Click the partner or third-party connection that you want to create a certificate for.
+. In the *Certificates* section, click *New*.
+. Select a certificate type.
+. Enter the settings for the selected certificate type:
++
+** <<public-certificate,Public Certificate>>
+** <<identity-key,Identity Key>>
++
+. Click *Save*.
+
+After you create a public certificate, you can assign it to the partner to use with AS2 Send to Partners endpoints. To do this, hover over the AS2 cell in the row containing the certificate and click *Assign*.
+
+Assigning a certificate to a partner unassigns the previously-assigned certificate.
+
+NOTE: You can also add certificates when you create endpoints. For more information, see xref:endpoint-https-send.adoc[], xref:endpoint-sftp-receive-target.adoc[], and xref:endpoint-sftp-send.adoc[].
+
+[[public-certificate]]
+=== Public Certificate Settings
+
+To configure settings for a public certificate:
+
+. In the *Certificate name* field, enter the name of the public certificate.
+. In the *Certificate* field, click *Choose file* and select a public certificate.
+
+[[identity-key]]
+=== Identity Key Certificate Settings
+
+To configure settings for an identity key certificate:
+
+. In the *Identity Key Name* field, enter the name of the identity key certificate.
+. In the *<partner> Identity Key PEM File* field, click *Choose file* and select an identity key certificate.
+. In the *Key passphrase* field, enter a passphrase for protecting the identity key certificate.
+. In the *Expiration date* field, optionally enter an expiration date for the certificate. This date is a reminder for you, but Anypoint Partner Manager doesn't enforce it.
+
+== Update a Certificate
+
+You can update a partner or third-party certificate by selecting a different certificate file to associate with the certificate:
```

Comment:
To update a partner or third-party certificate, select a different certificate file to associate with the certificate:

Reply:
Same comment as before.